[2023-08-29T16:00:17.000+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: vnstock-pipeline-1H.Vm_pub_pubsub scheduled__2023-08-29T15:00:00+00:00 [queued]>
[2023-08-29T16:00:17.005+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: vnstock-pipeline-1H.Vm_pub_pubsub scheduled__2023-08-29T15:00:00+00:00 [queued]>
[2023-08-29T16:00:17.006+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 4
[2023-08-29T16:00:17.022+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Vm_pub_pubsub> on 2023-08-29 15:00:00+00:00
[2023-08-29T16:00:17.026+0000] {standard_task_runner.py:57} INFO - Started process 203988 to run task
[2023-08-29T16:00:17.026+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2023-08-29T16:00:17.033+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'vnstock-pipeline-1H', 'Vm_pub_pubsub', 'scheduled__2023-08-29T15:00:00+00:00', '--job-id', '167', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmp4cktm5fc']
[2023-08-29T16:00:17.034+0000] {standard_task_runner.py:85} INFO - Job 167: Subtask Vm_pub_pubsub
[2023-08-29T16:00:17.074+0000] {task_command.py:415} INFO - Running <TaskInstance: vnstock-pipeline-1H.Vm_pub_pubsub scheduled__2023-08-29T15:00:00+00:00 [running]> on host vnstock-pipeline-vm.us-central1-c.c.vnstock-pipeline.internal
[2023-08-29T16:00:17.131+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='nguyentuanduong555@gmail.com' AIRFLOW_CTX_DAG_OWNER='Duong' AIRFLOW_CTX_DAG_ID='vnstock-pipeline-1H' AIRFLOW_CTX_TASK_ID='Vm_pub_pubsub' AIRFLOW_CTX_EXECUTION_DATE='2023-08-29T15:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-29T15:00:00+00:00'
[2023-08-29T16:00:18.620+0000] {logging_mixin.py:151} WARNING - /home/nguyentuanduong7/airflow/src/pluggin/stock_subscription.py:24 SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2023-08-29T16:00:18.631+0000] {logging_mixin.py:151} INFO -   Mã CP  Giá tham chiếu  Giá Trần  Giá Sàn  Giá khớp lệnh  Tổng Khối Lượng                              time
0   ACB         22050.0   23550.0  20550.0        22050.0         577470.0  2023-08-29 23:00:03.472337+07:00
1   TCB         34000.0   36350.0  31650.0        33900.0         370250.0  2023-08-29 23:00:03.472337+07:00
[2023-08-29T16:00:18.755+0000] {logging_mixin.py:151} INFO - publish message id 8306604433189938
[2023-08-29T16:00:18.820+0000] {logging_mixin.py:151} INFO - publish message id 8308784060946566
[2023-08-29T16:00:18.820+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-08-29T16:00:18.824+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=vnstock-pipeline-1H, task_id=Vm_pub_pubsub, execution_date=20230829T150000, start_date=20230829T160017, end_date=20230829T160018
[2023-08-29T16:00:18.861+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-29T16:00:18.872+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
